subcommand/mesh-init/command_test.go

```diff
@@ -972,6 +1009,24 @@ func constructContainerResponse(name, health string) awsutil.ECSTaskMetaContaine
 	}
 }
 
+// stopDataplaneContainer marks the dataplane container's status as STOPPED in the
+// task meta response
+func stopDataplaneContainer(taskMetadataResp *awsutil.ECSTaskMeta) {
+	index := -1
+	for i, c := range taskMetadataResp.Containers {
+		if isDataplaneContainer(c) {
+			index = i
+			break
+		}
+	}
+	if index == -1 {
```

Comment:
super minor nit: `if index >= -1` captures all possible invalid values for the index. I know it should never be anything other less than -1 but just a convention I am used to as it is a more constrained condition.

Reply:
Should this be index<=-1?

Reply:
Oops sorry typo. Yes `index <= -1`